### PR TITLE
add missing chainable/doc.

### LINKED ISSUE #6208: p5.Element doubleClicked missing @chainable in doc.
### Most appropriate sub-area of p5.js?

- [ ] Accessibility
- [ ] Color
- [X] Core/Environment/Rendering
- [ ] Data
- [X] DOM
- [ ] Events
- [ ] Image
- [ ] IO
- [ ] Math
- [ ] Typography
- [ ] Utilities
- [ ] WebGL
- [ ] Build Process
- [ ] Unit Testing
- [ ] Internalization
- [ ] Friendly Errors
- [ ] Other (specify if possible)

### p5.js version

1.6.0

### Web browser and version

1.52.122 Chromium: 114.0.5735.110

### Operating System

Linux

### Steps to reproduce this

https://github.com/processing/p5.js/blob/2474be4c7e76dad6fa9f7ae61b66aa9251f029b8/src/core/p5.Element.js#L247-L291

### PR DESCRIPTION
**Pull request by wuyudi**
Resolves #6208 , https://github.com/p5-types/p5.ts/issues/58

 Changes:

1. add `@chainable` to `doubleClicked` 's doc.
2. move/add some doc to properly place.

> It's strange that `p5.Element = class Element {` doesn't work but `p5.Element = class {` works.

 Screenshots of the change:

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by wuyudi** [comment]
fix mocha test by https://github.com/mochajs/mocha/issues/1588

File: test/unit/webgl/p5.Texture.js [hunk 1]
```diff
@@ -69,8 +72,9 @@ suite('p5.Texture', function() {
   };
 
   suite('p5.Texture', function() {
-    test('Create and cache a single texture with p5.Image', function() {
+    test('Create and cache a single texture with p5.Image', function(done) {
       testTextureSet(texImg1);
+      done();
```
Comment (davepagurek) [inline test/unit/webgl/p5.Texture.js:77]: Could you explain some more what this and the `beforeEach` is doing?

**Comment by wuyudi** [comment]
I don't understand.

https://github.com/processing/p5.js/pull/6209/commits/53c1f457c1a93189d8b5dd26801c4654821b571c 

why does this affect the result?

**Review by davepagurek — APPROVED** [review]
Thanks for adding these doc comments!

**Pull request merged**